t/request.t

```diff
@@ -553,3 +553,19 @@ Foo: bar
 Foo: baz, 123
 --- no_error_log
 [error]
+
+=== TEST 18: ngx.req.get_header (metatable is nil)
```

Comment:
need three blank lines before this line, you are recommended to use the `reindex` to reformated it automatically.